README.md

```diff
@@ -1 +1,5 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] Great Job Creating Readme.MD file, but the template has been updated and you need to use the [new one](https://github.com/microverseinc/readme-template), kindly update it and make sure to fill in the necessary fields 🙏

Reply:
Hi @ichala , so after making the required changes am I supposed to make another pull request or the one that already made will still be working for the upcoming changes?

Reply:
@Amen-Musingarimi keep working on this one , please do not close , merge or change the pull request until you get approved 🙏